### PR TITLE
Add off method

### PR DESCRIPTION
**Pull request by jamesfzhang**
What do you think? @peduarte See https://github.com/peduarte/wallop/issues/59 for details. Thanks!

### CODE REVIEW

**Comment by peduarte** [comment]
Hey @jamesfzhang i'm definitely opened to this. I'll do some tests to make sure it's working and then merge.

Thanks!

**Comment by jamesfzhang** [comment]
@peduarte Fantastic. I've been using this locally for testing and it works fine. Thanks!

File: js/Wallop.js [hunk 1]
```diff
@@ -154,9 +154,11 @@
 
   // Method so it is nicer for the user to use custom events
```
Comment (peduarte) [inline js/Wallop.js]: Can you replace this comment with:
```Method to bind custom event```

File: js/Wallop.js [hunk 2]
```diff
@@ -154,9 +154,11 @@
...
   WS.on = function (eventName, callback) {
-    this.$selector.addEventListener(eventName, function(event) {
-      return callback(event);
-    }, false);
+    this.$selector.addEventListener(eventName, callback, false);
+  };
+
+  WS.off = function (eventName, callback) {
```
Comment (peduarte) [inline js/Wallop.js]: Can you comment this method with:
```Method to unbind custom event```

File: js/Wallop.js (hunk 1 above)
Comment (jamesfzhang) [inline js/Wallop.js]: Sure

**Comment by jamesfzhang** [comment]
Just updated the commit with comments.

**Comment by peduarte** [comment]
Nice one, thank you

**Comment by peduarte** [comment]
@jamesfzhang Hey,
I did some tests, and in this scenario, the `off` method does not seem to work.
http://codepen.io/peduarte/pen/bEzVpb?editors=1111
Scroll down to the end of the code in the JS Panel and you'll see what it is.

**Comment by jamesfzhang** [comment]
@peduarte You need to use a named function (as opposed to an anonymous function) like so:

http://codepen.io/anon/pen/adXJVx?editors=1111

**Comment by peduarte** [comment]
Ok, that's good. I'm happy with it!
Can you also update the docs to include the `off` method there? Then it's good to merge :+1:

**Comment by jamesfzhang** [comment]
@peduarte Just updated the readme. Thanks!

**Comment by peduarte** [comment]
:+1:

**Pull request merged**